wca-regulations.md

```diff
@@ -413,11 +413,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A7a1) If the judge finds that the puzzle is solved, they call "OKAY".
         - A7a2) If the judge assigns any penalties, they call "PENALTY".
         - A7a3) If the result is DNF, the judge calls "DNF".
-    - A7b) The judge records the result on a score sheet.
+    - A7b) The judge records the result on a score sheet and signs next to the result.
```

Comment:
@timreyn, is us nationals even going to use cubecomps scorecards? As I recall, it never has in the past. In fact, the scorecards Tim generates for us Nats has the judge signature to the left of the competitor signature, so I think it would be great to remedy this discrepancy.